fastchat/model/model_adapter.py

```diff
@@ -662,6 +662,14 @@ def get_default_conv_template(self, model_path: str) -> Conversation:
         return base_adapter.get_default_conv_template(config.base_model_name_or_path)
 
 
+class DeepvkAdapter(BaseModelAdapter):
```

Comment:
Maybe it's better to use VK in naming instead of vk?

Reply:
`DeepVKAdapter`